src/oatpp-protobuf/Object.hpp

```diff
@@ -83,11 +83,11 @@ namespace __class {
   public:
 
     static oatpp::Type* getType(){
+      Type::Info info;
+      info.interpretationMap["protobuf"] = new Inter();
```

Comment:
It's a memory leak because `getType` method is called many times.
We want to put all this under `static Type type(...)` initialization in order to create only one instance of `Interpreter`.

Reply:
Fixed that so only one `Inter` instance is created for each Object class